src/main/java/com/spotinst/sdkjava/model/api/aws/managed/instance/ApiAmiBackup.java

```diff
@@ -0,0 +1,45 @@
+package com.spotinst.sdkjava.model.api.aws.managed.instance;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ApiAmiBackup {
```

Comment:
Did find the mapping construct for the class in the documentation. Need discussion.

Reply:
fixed